### ISSUE TITLE
capacity filter not working with incorrect format pk 

### ISSUE THREAD

**Issue by A-Harby** [threefoldtech/tfgrid-sdk-ts#88]
### Description

Some of the twins have an old incorrect format pk as '0x,' which does not work with deploying or capacity filtering and does not update like a null pk.

![image](https://user-images.githubusercontent.com/109473873/226897986-528ed16b-6f0e-4e62-877b-71c38bbd4ca3.png)

![image](https://user-images.githubusercontent.com/109473873/226898328-e4ec67b0-dc12-4a3c-a924-05a142555125.png)

**Comment by AhmedHanafy725** [comment]
a fix is done on the rmb direct client. it will be applied on weblets once it's released

**Comment by A-Harby** [comment]
Verified on Devnet,
I used twin with ID 291 which have a wrong pk as 0x.
![image](https://user-images.githubusercontent.com/109473873/234846620-321c7a68-2fcf-4f8f-ab6c-6cc318858166.png)
Once I logged into weblets and applied capacity filter, it works fine.
![image](https://user-images.githubusercontent.com/109473873/234846874-f53fe280-a9a9-458d-a060-06732a348810.png)
And pk was updated to correct value.
![image](https://user-images.githubusercontent.com/109473873/234846947-1b7cd826-230c-41cd-94b5-4828f732dc54.png)
